src/query/api/v1/handler/prometheus/native/common.go

```diff
@@ -240,12 +240,37 @@ func parseQuery(r *http.Request) (string, error) {
 	return queries[0], nil
 }
 
+func sanitizeSeries(series []*ts.Series) []*ts.Series {
```

Comment:
nit: Can we rename this perhaps just to `removeAllNaNsSeries(...)` or similar?  Not clear what sanitize does from just the name.

Reply:
sg

Reply:
Changed to `filterNaNSeries`